File: Daily Challanges/2026-05-May/2026-05-27.py
import math
def get_pizzas_to_order(hours_worked):
    slices = 0
    for hours in hours_worked:
        slices += max(2, math.ceil(hours/3))
    return math.ceil(slices / 8)

print(get_pizzas_to_order([8, 8, 8]))
print(get_pizzas_to_order([10, 9, 8, 2, 2, 6, 10]))
print(get_pizzas_to_order([1, 2, 3, 4, 5]))
print(get_pizzas_to_order([8, 8, 8, 8, 8, 8, 8, 8]))
print(get_pizzas_to_order([9, 9, 6]))
print(get_pizzas_to_order([9, 9, 6]))

File: Daily Challanges/2026-05-May/2026-05-27.js
function getPizzasToOrder(hoursWorked) {
  let slices = 0;
  for (const hours of hoursWorked) {
    slices += Math.max(2, Math.ceil(hours / 3));
  }
  return Math.ceil(slices / 8);
}

console.log(getPizzasToOrder([8, 8, 8]));
console.log(getPizzasToOrder([10, 9, 8, 2, 2, 6, 10]));
console.log(getPizzasToOrder([1, 2, 3, 4, 5]));
console.log(getPizzasToOrder([8, 8, 8, 8, 8, 8, 8, 8]));
console.log(getPizzasToOrder([9, 9, 6]));
console.log(getPizzasToOrder([10, 12, 16, 9, 8, 11, 15, 8, 0]));
